pyproject.toml

```diff
@@ -93,7 +93,7 @@ docs = [
 ]
 linting = [
     "eval-type-backport>=0.1.3",
-    "ruff==0.1.3",
+    "ruff==0.2.1",
```

Comment:
Is this the only line that changed? Did the rest just get reordered?

Reply:
Yes, apart from the version bump I also updated the ruff config as a new explicit `lint` section was introduced since `0.2.0`.

The diff is a bit messy and the Github syntax highlighting issue is a bit annoying (considering it's been like this for a couple months now).